coretex/qiime2/utils.py

```diff
@@ -107,3 +141,7 @@ def getDenoisedSamples(dataset: CustomDataset) -> List[CustomSample]:
 
 def getPhylogeneticTreeSamples(dataset: CustomDataset) -> List[CustomSample]:
     return dataset.getSamples(isPhylogeneticTreeSample)
+
+
+def getMetadataSamples(dataset: CustomDataset) -> CustomSample:
+    return dataset.getSamples(isMetadataSample)
```

Comment:
getSamples function returns an array, not an CustomSample object. There should be only one metadata sample inside the dataset